ui/app/src/components/workspaces/WorkspaceOperationsPanel.tsx

```diff
@@ -0,0 +1,100 @@
+import { DefaultPalette, IStackItemStyles, IStackStyles, Stack } from "@fluentui/react";
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Operation } from '../../models/operation';
+import { Workspace } from '../../models/workspace';
+import { ApiEndpoint } from '../../models/apiEndpoints';
+import { HttpMethod, ResultType, useAuthApiCall } from '../../useAuthApiCall';
+import config from '../../config.json';
+
+
+
+interface WorkspaceOperationPanelProps {
+    workspace: Workspace
+}
+
+interface WorkspaceOperationPanelItemProps {
+    header: String,
+    val: String
+}
+
+const WorkspaceOperationPanelItem: React.FunctionComponent<WorkspaceOperationPanelItemProps> = (props: WorkspaceOperationPanelItemProps) => {
+
```

Comment:
question: Is three-blank-line spacing normal here? I'd have expected just one blank line between function sig and body, but I honestly don't know the convention for .ts. Either way, it's not consistent further down (2 new lines).

(if we have a linter set up, and this is linted, please ignore me 😄 )

Reply:
Yep, linter will definitely structure it. It's fixed for this PR though!